backend/apps/sandbox/serializers.py

```
@@ -255,3 +255,51 @@ class Meta:
+    class Meta:
+        model = TriageIssue
+        fields = [
+            "id",
+            "title",
+            "raw_issue_title",
+            "raw_issue_body",
+            "correct_labels",
+            "hint",
+            "difficulty",
+            "created_at",
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)